src/bot.coffee

```diff
@@ -180,7 +180,7 @@ class SlackBot extends Adapter
 
       when 'channel_topic', 'group_topic'
         @robot.logger.debug "#{user.name} set the topic in #{channel.name} to #{topic}"
-        @receive new TopicMessage user, message.topic, message.ts
+        @receive new TopicMessage user, message.setTopic, message.ts
```

Comment:
@DEGoodmanWilson looks like you might have a search and replace mistake. this line should not change.